external/chromium/src/base/allocator/partition_allocator/thread_isolation/thread_isolation.cc

```diff
@@ -37,7 +37,7 @@ void WriteProtectThreadIsolatedMemory(ThreadIsolationOption thread_isolation,
 #elif BUILDFLAG(ENABLE_MTE_ISOLATION)
   partition_alloc::internal::TagMemoryWithMte(
       thread_isolation.enabled ? thread_isolation.pkey : kDefaultPkey, address,
-      size);
+      (size+0xfff)&~0xfff);
```

Comment:
Can we use PAGE_SIZE if it's defined?

Reply:
PAGE_SIZE is not defined here, but it would be cleaner to use it and a ROUND_UP macro. I don't know if it's worth the trouble to redefine them or factor out an include we could use for this one occurrence.